aws-auth-cognito/src/test/java/com/amplifyframework/auth/cognito/StateTransitionTests.kt

```diff
@@ -300,15 +308,106 @@ class StateTransitionTests : StateTransitionTestBase() {
                             AuthenticationEvent.EventType.SignInRequested(
                                 "username",
                                 "password",
-                                AuthSignInOptions.defaults()
+                                AWSCognitoAuthSignInOptions.builder()
+                                    .metadata(mapOf())
+                                    .authFlowType(AuthFlowType.USER_SRP_AUTH)
+                                    .build()
                             )
                         )
                     )
                 }
+                val authNState = it.authNState.takeIf { itN ->
```

Comment:
`authNState is AuthenticationState.SignedIn && authZState is AuthorizationState.SessionEstablished` represents the correct signed in state.